<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>文泉驿微米黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/resource/bg.png);
}
#centralwidget *{
border-image: url();
}
</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>86</y>
      <width>1560</width>
      <height>900</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1560</width>
        <height>900</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="page_layout" rowstretch="0" columnstretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="ver_label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>988</y>
      <width>200</width>
      <height>92</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ip_label">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>988</y>
      <width>200</width>
      <height>92</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="ev_list" native="true">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>20</y>
      <width>360</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>988</y>
      <width>200</width>
      <height>92</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/xiangqianxuanze.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/xianghouxuanze.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="matrix_1_button">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/1-gongge.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="matrix_4_button">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/4-gongge.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="matrix_16_button">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/16-gongge.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>1010</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/setup.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
